packages/ecs-agent/ecs.config

```diff
@@ -39,3 +39,6 @@ ECS_BACKEND_HOST="{{settings.ecs.backend-host}}"
 {{#if settings.ecs.awsvpc-block-imds}}
 ECS_AWSVPC_BLOCK_IMDS="{{settings.ecs.awsvpc-block-imds}}"
 {{/if}}
+{{#if settings.ecs.enable-container-metadata}}
+ECS_ENABLE_CONTAINER_METADATA="{{settings.ecs.enable-container-metadata}}"
```

Comment:
@arnaldo2792 Since this adds a new setting, don't forget to make an issue on bottlerocket/bottlerocket-project-website once meged.